### DISCUSSION TITLE
Code review IM-13 Potapenko

### DISCUSSION THREAD

**Discussion by m1guelten** [HowProgrammingWorks/LiveQA#70] [Code review]
1 курс 2 семестер

Пишу браузерну гру "CHECKERS RACES".
Суть гри:
«CHECKERS RACES» - це гра, в якій шашки на випередження пробують зайняти місця в своєму логові. Перед тим як потрапити в логово, кожна шашка повинна пройти повне коло.
Буду вдячний за рев'ю [проекта](https://github.com/m1guelten/kursova.git)

**Comment by tshemsedinov** [comment]
See 1st review; https://github.com/m1guelten/kursova/issues/1

**Comment by m1guelten** [comment]
1 курс 2 семестер

Пишу браузерну гру "CHECKERS RACES".
Врахував всі ваші зауваження!
Буду вдячний за нове рев'ю [проекта](https://github.com/m1guelten/kursova.git)
